.github/workflows/package.yaml

```diff
@@ -45,7 +45,7 @@ jobs:
         python-version: '3.x'
     - uses: actions/setup-go@v4
       with:
-        go-version: '^1.18'
+        go-version: '^1.21'
```

Comment:
So we have other projects under src/go/**/go.sum that are still using go 1.18, what does this mean for the other projects?